src/test/test.spec.js

```diff
@@ -0,0 +1,235 @@
+import chaiHttp from 'chai-http';
+import chai, { expect } from 'chai';
+import { describe, it } from 'mocha';
+import app from '../app';
+
+chai.use(chaiHttp);
+
+describe('SendIT API TESTS', () => {
```

Comment:
Try separating your tests into different files. You can have a file for testing get requests for a parcel, and the other file to test for POST requests and a final file to test for PUT requests. You may name them as so; `retrieve_parcels_test.js`, `create_parcels_test.js`, `update_parcles_test.js`

Reply:
Done

Reply:
Done